benchmark/benchmark_hydrostatic_model.jl

```diff
@@ -55,13 +59,13 @@ Architectures = has_cuda() ? [CPU, GPU] : [CPU]
 grid_types = [
     :RegularRectilinearGrid,
     :RegularLatitudeLongitudeGrid,
-    :ConformalCubedSphereFaceGrid,
+    #:ConformalCubedSphereFaceGrid,
```

Comment:
?

Reply:
Hmm, the issue is that these benchmarks were written so that the test with `ConformalCubedSphereFaceGrid` (comprising a single face of the cubed sphere) has the same _total size_ as the test with the full `ConformalCubedSphereGrid` (which comprises six faces). For this we have to be able to construct arbitrary `ConformalCubedSphereFaceGrid`. Except that we can't; the constructor for arbitrary sized conformal cube face grids returns zeros for all the grid metrics except coordinates. We can only build them from file right now. Because of this I didn't bother to adapt that constructor to the GPU (what's the point of doing a bunch of work for an unuseable piece of code?) But as a result we can't run this benchmark.

Reply:
gotcha; perhaps add small comment next to the commented line if we feel that merging this with a commented line is the way forward?

Reply:
Done. I think @ali-ramadhan might be updating / playing around with these benchmarks in not too long, too.